src/bindings/swift/gen_swift.rs

```diff
@@ -0,0 +1,498 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use anyhow::Result;
+use askama::Template;
+
+use crate::interface::*;
+
+// Some config options for it the caller wants to customize the generated python.
+// Note that this can only be used to control details of the python *that do not affect the underlying component*,
+// sine the details of the underlying component are entirely determined by the `ComponentInterface`.
+pub struct Config {
+    // No config options yet.
+}
+
+impl Config {
+    pub fn from(_ci: &ComponentInterface) -> Self {
+        Config {
+            // No config options yet
+        }
+    }
+}
+
+#[derive(Template)]
+#[template(
+    ext = "h",
+    escape = "none",
+    source = r#"
+// This file was autogenerated by some hot garbage in the `uniffi` crate.
+// Trust me, you don't want to mess with it!
```

Comment:
I'm very happy that you chose to copy this comment across verbatim 😆

Reply:
https://twitter.com/jckarter/status/1141752551185866753 🤣

Reply:
If this project goes anywhere, I'd like a t-shirt with "hot garbage" on it.